RELEASE_NOTES.md

```diff
@@ -44,39 +44,44 @@ the following platforms:
  * Experimental support for Zephyr RTOS (`-target zephyr_stm32f4_disco`) is available on macOS, Linux
```

Comment:
We can stop mentioning this one as well.
```suggestion
```

Reply:
Why do you think so? Sounds somewhat premature.

Reply:
1. We don't even know is it working :)
2. Even if it does, UX is just terrible.
I'm not suggesting to drop Zephyr support from compiler. Just to stop attracting users with the feature that is barely usable.

Reply:
Made the mention more explicit instead of removing it.